addons/web_unsplash/static/src/xml/unsplash_image_widget.xml

```diff
@@ -62,7 +62,7 @@
                     Search is temporarily unavailable
                 </h4>
                 <div class="details">
-                    The max number of searches is exceeded. Please retry in an hour or extend to a better account.
+                    The maximum number of searches is exceeded. Please retry in a few minutes.
```

Comment:
@JKE-be Quid translations?

Reply:
in this case, we keep old translation since there are no more value.

My answer from yesterday was for wrong text. I prefer correct text in english that wrong information translated ;)